linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/rule.yml

```diff
@@ -7,9 +7,15 @@ title: 'Encrypt Audit Records Sent With audispd Plugin'
 description: |-
     Configure the operating system to encrypt the transfer of off-loaded audit
     records onto a different system or media from the system being audited.
+{{% if product in ["rhel8", "fedora"] %}}
+    Uncomment the <tt>transport</tt> option in <pre>/etc/audit/audisp-remote.conf</pre>,
```

Comment:
In file `/etc/audit/auditd.conf` `transport` option is already set as `TCP`.

Reply:
You're right it's not commented.

Reply:
In `/etc/audit/audisp-remote.conf `, `transport` option is not commented also.